modules/auxiliary/gather/kerberos_enumusers.rb

```diff
@@ -39,10 +40,9 @@ def initialize(info = {})
   end
 
   def user_list
-    users = nil
     if File.readable? datastore['USER_FILE']
-      users = File.new(datastore['USER_FILE']).read.split
```

Comment:
User accounts with spaces weren't be handled correctly, I've changed this to `readlines` - but long term it should be using the brute force support for running in parallel